Bottom: 10b638f6a688c12b82c41f2ba10abde8bd58656d
Top:    ca6e3685bbba527de35f381b9b4f3e7846826f9f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 02:59:24 +0100

Change scroll-wheel bindings to match more GTK applications










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 9b6e612..3b0b0d4 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
